app/frontend/Settings/Mail/Manual/ManualConfigURL.svelte

```diff
@@ -29,6 +29,9 @@
     <hbox class="authMethod" class:error={authError}>
       <select bind:value={config.authMethod} required>
         <option value={AuthMethod.Password}>{$t`Password`}</option>
+        {#if $config.protocol == "ews"}
```

Comment:
Any reason to use `$config` here and regular `config` two lines above? (Rest of the PR seems fine.)